packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -269,6 +269,13 @@ class DraggableScrollableController extends ChangeNotifier {
 /// scrolled up or down, until they reach the top of the list again and the user
 /// drags the sheet back down.
 ///
+/// On Desktop and Web platforms, the vertical drag gesture is disabled by default
```

Comment:
Similar to other sample code we have updated for desktop and web (like https://github.com/flutter/flutter/pull/135910) instead of documenting that it does not work on those platforms, we should make it work by updating the sample to show common design patterns developers would use in real world applications.
For this one, it is common for folks to add a drag handle to allow the sheet to be dragged open with a mouse, or a button that pops the sheet open and closed.

Reply:
Hi @Piinks :) 
Thanks for your review. I thought this was just an issue about docs so I just added comments at PR only.

However, after considering your suggestions, I prefer the drag handle option because it's neat. I've implemented the following change, let me know if it's good enough:

<details>
<summary>Demo video</summary>

| light | dark |
| --------------- | --------------- |
<video src="https://github.com/flutter/flutter/assets/104349824/6fe5728d-a114-437b-afaf-e27b8861c7ff"/> | <video src="https://github.com/flutter/flutter/assets/104349824/0e2003ca-cf9b-41b9-9026-6ea2e7ea419d"/>

</details>

Additionally, I see that the old sample code is not previewable because it is being wrapped in a `{@tool snippet}` tag instead of `{@tool dartpad}` like other APIs. Would you like me to make this update in the PR as well? If so, I would probably separate the example code file and of course, need a test for the example :)

Reply:
That would be superb! Thank you! 🎉